### PR TITLE
[CIS-266] Update SwiftLint and SwiftFormatter

### PR DESCRIPTION
**Pull request by buh**
- updated code in `v2` by linting
- updated `.swiftformat` file
- added SwiftLint for `v3`
- updated code for `v3` by linting and formatting
- updated `bootstrap.sh` to install a specific version of `swiftformat`

#skip_changelog

### CODE REVIEW

File: Sources/.swiftlint.yml [hunk 1]
```diff
@@ -46,17 +47,16 @@ identifier_name:
...
   ignores_comments: true
+  ignores_urls: true
 
 cyclomatic_complexity:
   ignores_case_statements: true
   warning: 25
   error: 30
 
-file_length: 510
+file_length: 500
```
Comment (VojtaStavik) [inline Sources/.swiftlint.yml:60]: I'd suggest removing this. Sometimes it's helpful to have longer files. I'd leave this at the developer's opinion.

File: Sources_v3/Controllers/ChannelController_Tests.swift [hunk 2]
```diff
@@ -606,4 +606,4 @@ private class TestDelegateGeneric: QueueAwareDelegate, ChannelControllerDelegate
 private struct TestMemberEvent: MemberEvent, Equatable {
     let cid: ChannelId
     let userId: UserId
-}
+} // swiftlint:disable:this file_length
```
Comment (VojtaStavik) [inline Sources_v3/Controllers/ChannelController_Tests.swift:609]: exactly, get rid of this rule completely please. It doesn't make sense.

File: Sources_v3/WebSocketClient/WebSocketClient_Tests.swift [hunk 3]
```diff
@@ -245,7 +245,7 @@ class WebSocketClient_Tests: StressTestCase {
     
     func test_webSocketPingController_connectionStateDidChange_calledWhenConnectionChanges() {
         test_connectionFlow()
-        AssertAsync.willBeEqual(pingController.connectionStateDidChange_connectionStates,
+        AssertAsync.willBeEqual(pingController.connectionStates,
```
Comment (VojtaStavik) [inline Sources_v3/WebSocketClient/WebSocketClient_Tests.swift:248]: These variable names follow specific conventions. It's the `<name of the function>_<recorded value>`. Please don't randomly change this.

Comment (buh) [reply Sources_v3/WebSocketClient/WebSocketClient_Tests.swift:248]: It was really long name, more then 40 characters and I think it's clear this is a collection connection states.

File: Sources/.swiftlint.yml (hunk 1 above)
Comment (VojtaStavik) [reply Sources/.swiftlint.yml:60]: @buh ☝️ ??

File: bootstrap.sh [hunk 4]
```diff
@@ -4,8 +4,9 @@
 # You should have homebrew installed.
 # If you get `zsh: permission denied: ./bootstrap.sh` error, please run `chmod +x bootstrap.sh` first
 
-# Install swiftformat
-brew install swiftformat
+# Install swiftformat 0.45.6
+brew unlink swiftformat
+brew install https://raw.githubusercontent.com/nicklockwood/homebrew-core/0269354fff817670543922c73bb522adca3cc46b/Formula/swiftformat.rb
```
Comment (VojtaStavik) [inline bootstrap.sh:9]: This should install the version of swiftformat we want.

Comment (VojtaStavik) [reply bootstrap.sh:9]: We should add Swiftlint here, too

File: Sources/.swiftlint.yml (hunk 1 above)
Comment (buh) [reply Sources/.swiftlint.yml:60]: I think this value doesn’t work because I excluded the rule.

File: bootstrap.sh (hunk 4 above)
Comment (buh) [reply bootstrap.sh:9]: We already have it with: `bundle install` at the end of `bootstrap.sh`

Comment (VojtaStavik) [reply bootstrap.sh:9]: Nice!

**Pull request merged**